client/src/utils.rs

```diff
@@ -65,7 +65,7 @@ pub fn send_and_wait_for_in_block<C: Encode>(api: &Api, xt: EncointerXt<C>) -> O
 }
 
 pub fn send_xt_hex_and_wait_for_in_block(api: &Api, xt_hex: String) -> Option<H256> {
-	ensure_payment(&api, &xt_hex);
+	ensure_payment(&api, &xt_hex, None);
```

Comment:
I know it is a bit tedious, but I would like to have the payment_cid arg passed here too.